tests/functional/exporters/testing/test_json.py

```diff
@@ -60,8 +58,8 @@ def test_json_exporter(tmpdir):
 
 def test_implicit_exporter_initialization(tmpdir):
     """
-        An implicit JSON should be generated if `json_path` is available
-        via cmdline args but no exporters were declared programmatically.
+    An implicit JSON should be generated if `json_path` is available
+    via cmdline args but no exporters were declared programmatically.
     """
     json_path = tmpdir.mkdir('reports').join('report.json').strpath
 
```

Comment:
Please make these changes in a separate review if they are not functionally part of the http exporter

Reply:
It's a non-functional small change so I modified it by the way. maybe next we can collect these thing and made a PR including all of them